lib/proxy/log_buffer/decorator.rb

```diff
@@ -10,17 +10,15 @@ def initialize(logger, buffer = Proxy::LogBuffer::Buffer.instance)
 
     def add(severity, message = nil, progname = nil, backtrace = nil, a_module = 'core', &block)
```

Comment:
I think @witlessbird made a good point here - it even incurs a serious performance penalty. If you're not calling block explicitely, just remove it IMO - http://mudge.name/2011/01/26/passing-blocks-in-ruby-without-block.html

Where is this defined in Ruby core? If `block` is defined but doesn't need to be passed because `yield` is used, submit a patch to fix the performance issue :smile: